src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/CollectionDataContract.cs

```diff
@@ -1524,17 +1529,21 @@ internal bool RequiresMemberAccessForWrite(SecurityException securityException)
             return false;
         }
 
-        public override void WriteXmlValue(XmlWriterDelegator xmlWriter, object obj, XmlObjectSerializerWriteContext context)
+        public override void WriteXmlValue(XmlWriterDelegator xmlWriter, object obj, XmlObjectSerializerWriteContext? context)
```

Comment:
should obj be nullable?

Reply:
See https://github.com/dotnet/runtime/pull/41476#discussion_r479391298.